components/net/Cargo.toml

```diff
@@ -47,6 +47,7 @@ percent-encoding = { workspace = true }
 pixels = { path = "../pixels" }
 profile_traits = { workspace = true }
 rayon = { workspace = true }
+rkv = "0.10"
```

Comment:
You should be able to de-duplicate `syn` by upgrading to the most recent version of `rkv` instead of using this one, which is 5 years old.

Reply:
Switching to heed instead of rkv, so this should be resolved.